content/blog/2022-05-04-e2e-cv-pt2.md

```diff
@@ -0,0 +1,197 @@
+---
+title: 'End-to-End Computer Vision API, Part 2: Local Experiments'
+date: 2022-05-04
+description: >
+  In part 1 of this series of posts, we introduced a solution to a common
+  problem faced by companies in the manufacturing industry: detecting defects
+  from images of products moving along a production line. The solution we
+  proposed was a Deep Learning-based image segmentation model wrapped in a web
+  API. We talked about effective management and versioning of large datasets and
+  the creation of reproducible ML pipelines.
+
+descriptionLong: |
+  _This is the second part of a three-part series of posts._
+
+  In part 1 of this series of posts, we introduced a solution to a common problem faced by companies in the manufacturing industry: detecting defects from images of products moving along a production line. The solution we proposed was a Deep Learning-based image segmentation model wrapped in a web API. We talked about effective management and versioning of large datasets and the creation of reproducible ML pipelines.
+
+picture: 2022-05-04/e2e-cv-pt2-cover.png
+author: alex_kim
+commentsUrl: https://discuss.dvc.org/t/end-to-end-computer-vision/1178
```

Comment:
Wondering if we should have a separate topic for each?  Not sure it's necessary though since this is an interconnected series.  I think it's ok to leave like this.  We rarely get comments on blog posts.

Reply:
Since all parts are closely related, I think it's better to have one place for potential comments.